.github/workflows/tii-depthai-ctrl.yaml

```diff
@@ -1,20 +1,81 @@
 name: tii-depthai-ctrl
 
 on:
-  repository_dispatch:
-    types: [fog-ros-baseimage-update]
   push:
+  pull_request:
   workflow_dispatch:
+    inputs:
+      build_amd64:
+        description: 'Build for AMD64?'
```

Comment:
Is the '?' a typo?

Reply:
It is somewhat. It was supposed to appear as a question during workflow dispatch selections, but doesn't really matter. I will remove them for better clarity